Annotations/Arrow/ArrowDemo/ArrowDemo_NET.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net8.0-windows</TargetFramework>
+    <UseWPF>true</UseWPF>
+  </PropertyGroup>
```

Comment:
<img width="547" alt="image" src="https://github.com/user-attachments/assets/b2e5c3ee-e86d-45d8-9ee5-a98ba373b733">

You have updated that it throws error. Then why it was not added here? Why that error is not thrown here?

Reply:
Two samples have only that issue soo we not update that here